src/commands/generate-path/path.generate.ts

```diff
@@ -101,48 +77,39 @@ export const commandGeneratePath = command({
  * @returns
  */
 export function generatePath(metadata: PathMetadata): string {
-  const name = formatName(metadata.region, metadata.geographicDescription);
-  const surveyName = metadata.date ? `${name}_${metadata.date}` : name;
-
-  if (metadata.category === dataCategories.SCANNED_AERIAL_PHOTOS) {
+  if (metadata.geospatialCategory === GeospatialDataCategories.ScannedAerialPhotos) {
     // nb: Historic Imagery is out of scope as survey number is not yet recorded in collection metadata
-    throw new Error(`Automated target generation not implemented for historic imagery`);
-  }
-
-  if ([dataCategories.URBAN_AERIAL_PHOTOS, dataCategories.RURAL_AERIAL_PHOTOS].includes(metadata.category)) {
-    return `s3://${metadata.targetBucketName}/${metadata.region}/${surveyName}_${metadata.gsd}m/rgb/${metadata.epsg}/`;
+    throw new Error(`Historic Imagery ${metadata.geospatialCategory} is out of scope for automated path generation.`);
   }
 
-  if (metadata.category === dataCategories.SATELLITE_IMAGERY) {
-    return `s3://${metadata.targetBucketName}/${metadata.region}/${surveyName}_${metadata.gsd}m/rgb/${metadata.epsg}/`;
+  if (
+    (
+      [
+        GeospatialDataCategories.UrbanAerialPhotos,
+        GeospatialDataCategories.RuralAerialPhotos,
+        GeospatialDataCategories.SatelliteImagery,
+      ] as string[]
```

Comment:
I think the typing here is off ?

this shouldn't need a cast. Is something set as a "string" when it should be a `GeospatialDataCategories`

Reply:
Do you have a more specific suggestion? `as typeof GeospatialDataCategories[]` doesn't work:

> Argument type string is not assignable to parameter type {readonly SatelliteImagery: "satellite-imagery", readonly AerialPhotos: "aerial-photos", readonly RuralAerialPhotos: "rural-aerial-photos", readonly UrbanAerialPhotos: "urban-aerial-photos", readonly Dem: "dem", readonly ScannedAerialPhotos: "scanned-aerial-photos", readonly Dsm: "dsm"}

I also can't use `metadata.geospatialCategory as typeof GeospatialDataCategories`:

> Argument type {readonly SatelliteImagery: "satellite-imagery", readonly AerialPhotos: "aerial-photos", readonly RuralAerialPhotos: "rural-aerial-photos", readonly UrbanAerialPhotos: "urban-aerial-photos", readonly Dem: "dem", readonly ScannedAerialPhotos: "scanned-aerial-photos", readonly Dsm: "dsm"} is not assignable to parameter type "urban-aerial-photos" | "rural-aerial-photos" | "satellite-imagery"

Reply:
I think it was mostly caused by type narrowing of the arrays, it is generally much more efficent to use `||` if statements than `.includes` They also fix the typing problem, 

I have pushed a commit to fix it.